bin/crew

```diff
@@ -1493,6 +1493,7 @@ def print_deps_tree(args)
 end
 
 def upload(pkg_name = nil, pkg_version = nil)
+  abort "\nPackage to be uploaded was not specified.\n".lightred if pkg_name.nil?
```

Comment:
There is already logic a few lines down that loops through all binaries and uploads all if no package argument(s) are provided.  I think the issue is it aborts if there are no .tar.xz binaries.

Reply:
This still looks for the right binaries though, no?

Reply:
This code is there to abort early if no pkg_name is specified, as when one just invokes `crew upload` without anything else.

The code later on separately looks for whether binaries are available.

Reply:
I don't think you understand how this was originally intended to work.  If no package arguments are provided then it should still upload all the binaries it can find in the release directories.